docs/team/justinhoe.md

```diff
@@ -5,23 +5,48 @@ title: Justin Hoe's Project Portfolio Page
 
 ### Project: Connections
 
-Connections is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+Connections is a desktop address book application used for teaching Software Engineering principles. 
+The user interacts with it using a CLI, and it has a GUI created with JavaFX. 
+It is written in Java, and has about 10 kLoC.
 
 Given below are my contributions to the project.
 
-* **New Feature**: 
-
+* **New Feature**: Added a Birthday Reminder List
+    * The display contacted sorted by birthdays in ascending order, starting with the 
+      nearest upcoming birthday all the way through the latest birthday and then looping back 
+      to the earliest birthday.
+    * The list is responsive to changes in contact details be it add, delete or edit command. The list 
+      reorders contacts where appropriate to ensure it stays sorted as described above.
+    * Birthday at present day and coming within 7 days are color coded as well as personalised message.
```

Comment:
`Birthday at present day and upcoming birthdays within 7 days are color coded with personalised message.` would be better?